dist/i18n/GUI_Strings_es.txt

```diff
@@ -61,8 +61,6 @@ menu.edit.find = Buscar...
 menu.edit.findNext = Buscar siguiente...
 menu.edit.shiftLeft = Des-Sangrar texto
 menu.edit.shiftRight = Sangrar texto
-menu.edit.comment = Comentarios al texto
-menu.edit.uncomment = Descomentar
```

Comment:
Instead of losing these translations altogether, can we do just say "Comment" / "Uncomment". For instance here we could add `menu.edit.toggleComment = Comentarios al texto / Descomentar`, and similarly for the other files.